content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+---
+author: erizocosmico
+date: 2017-07-10
+title: "source{d} tech talks, frontend series"
+image: /post/frontend-talks/header-pic.jpg
+description: "Once every few months, source{d} organizes small conferences around a very specific topic. On June, 24th the topic was frontend and the talks were hosted in our Madrid offices. Almost 50 people joined us for a day full of things to learn about frontend technologies."
+categories: ["technical", "frontend"]
+---
+
+![Schedule with our logo](/post/frontend-talks/header-full.jpg)
+
+Once every few months, source{d} organizes small conferences around a very specific topic. On June, 24th the topic was frontend and the talks were hosted in our Madrid offices. Almost 50 people joined us for a day full of things to learn about frontend technologies.
+
+The topics of the talks were very diverse, from animations with Ember to multi-threaded web to React styiling to accessibility. Since frontend is such a wide area in programming, we aimed to provide talks from a lot of different topics, which we thing we accomplished.
+
+## Conferences are no fun on an empty stomach
+
+First things first, we started the day with a nice welcome breakfast for all the attendees and speakers in which people could talk and get to know each other as well as the speakers before their talks.
+
+## Opening speech
+
+After everyone was in their seat, [Margarida García](https://twitter.com/margaridagsl) gave the opening talk and welcomed all the people in the room. She talked a bit about what we do here at source{d} and explained what the source{d} tech talks was about and why the community was so important to us.
```

Comment:
Garcia sin tilde :)

welcomed everyone in the room. ... explained what our tech talks are about and why these and other initiatives we hold to bring together a strong community are so important to us.